post_processing/compare_outputs.jl

```diff
@@ -63,12 +65,15 @@ compare(a::F, b::F, pn0 = "") where {F <: Fields.Field} =
     compare(true, a, b, pn0)
 function compare(zero_error, a::F, b::F, pn0 = "") where {F <: Fields.Field}
     isempty(propertynames(a)) || return _compare(zero_error, a, b, pn0)
-    err = abs.(Array(parent(a)) .- Array(parent(b)))
-    if !(maximum(err) == 0.0)
+    abs_err = abs.(Array(parent(a)) .- Array(parent(b)))
+    rel_err =
+        norm(Array(parent(a)) .- Array(parent(b))) ./ norm(Array(parent(a)))
```

Comment:
any reason we're doing this on the arrays rather than the fields? @charleskawczynski ?